daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -542,7 +547,7 @@ private[lf] class Runner(
       case "daml-script" =>
         new v1.Runner(this).runWithClients(initialClients, traceLog, warningLog, canceled)
       case "daml3-script" =>
-        new v2.Runner(this).runWithClients(initialClients, traceLog, warningLog, canceled)
+        new v2.Runner(this, initialClients, traceLog, warningLog, canceled).getResult()
```

Comment:
We _could_, I'm trying to avoid changing much about v1 here, but I guess the v2.Runner is already a fair bit of change.
My view on this is that v1 will be purged from this earth, so improving it is wasted time